### ISSUE TITLE
Dimention card not working when added to the wireless transmittor

### ISSUE THREAD

**Issue by C1oudsu** [Hexeption/RSInfinityBooster#29]
The normal range upgrades work but when i add the dimension card it does nothing

**Comment by BloodyMikey** [comment]
In what transmitter did you place it

**Comment by Zarkazm** [comment]
I realize this is an old open issue, but I believe the original poster was referring to the Advanced Wireless Transmitter from the Reborn Storage addon mod.  In both All The Mods 8 and 9 packs, I've observed that the Infinite and Dimension cards are treated as normal range boosters in the Advanced Wireless Transmitter, while working as expected in the normal Wireless Transmitter from the base Refined Storage mod.

**Comment by jollyfish91** [comment]
not enough energy mauve ?

**Comment by ShadowFauxlFoxl** [comment]
Even now, its still an issue. Placed the card in a normal transmitter, and still no dimensional range

**Comment by icram0719** [comment]
> Even now, its still an issue. Placed the card in a normal transmitter, and still no dimensional range

have the same problem

**Comment by AdamLeyshon** [comment]
Yep, I'm running 1.20.1 with Forge 47.3.10 and I've tried both the Normal and Advanced Wireless transmitters with both Wireless Crafting grid and Super Wireless Crafting grid from StorageReborn, just get the standard "No network transmitters in range" message.

Both tranmistters show the max range and draw 8000 or 12000 FE/t when using both cards, so it definitely registers that they're in place.

**Comment by Teros123** [comment]
> **[icram0719](/icram0719) ** commented

Force load the chunk your storage is in. seems to work for me

**Comment by ItzCrashzzz** [comment]
> > **[icram0719](/icram0719) ** commented
> 
> Force load the chunk your storage is in. seems to work for me

It still doesnt work, i even tried multiple minecraft versions and still nothing, from 1.20.1 to 1.21.1 the dimension card seems to not work on either of the transmiters, loaded or unloaded chunks

**Comment by icram0719** [comment]
in 1.21.1 it just straight up does not work or i am doing something wrong, those are the only 2 options, i have the chunks force loaded and have a 100K/tick energy generation and it still does not work, and you cant look at the controller anymore to see the power usage of the different parts so you don't even know what it uses but if 100k is not enough for it then the uninstall button may be enough idk, its really annoying when you finally make something so expensive and it just straight up doesn't work, same with the normal infinite range one.

**Comment by Grant-K** [comment]
The new RE2 and these cards do not work by default because the new controller holds only 1kfe and these draw 4k and 8k respectively so it draws too much power at this time set it below 100 in config and it fixes

**Comment by yMemez** [comment]
Problem still persists. On 1.20.1 Forge, the range seems to work fine but the cross-dimension access is not.

When not in the same dimension as the wireless transmitter with the Dimensional Card, and using the `/refinedstorage network get [dimension] [coordinates of controller]` command, it tells me the position of the controller is not loaded - even though I use chunk loaders to load the respective chunks. 

Another command I used - `/refinedstorage network list [dimension]` doesn't give any error, and provides the Mean Tick Time and Mean TPS of the controller and keeps updating the numbers upon re-using the command - and the other command tells me the position isn't loaded.

# TL:DR - Possible Solution
I found [this issue](https://github.com/TechReborn/RebornStorage/issues/211) on Reborn Storage saying that the mod isn't compatible with RSInfinityBooster. After removing it from my mod list and testing it out I can confirm that this is true. 

You can replace Reborn Storage with the [ExtraStorage](https://www.curseforge.com/minecraft/mc-mods/extrastorage/comments) mod that adds storage disks with increased storage and faster autocrafters with increased pattern slots.

**Comment by Mailootje** [comment]
> Problem still persists. On 1.20.1 Forge, the range seems to work fine but the cross-dimension access is not.
> 
> When not in the same dimension as the wireless transmitter with the Dimensional Card, and using the `/refinedstorage network get [dimension] [coordinates of controller]` command, it tells me the position of the controller is not loaded - even though I use chunk loaders to load the respective chunks.
> 
> Another command I used - `/refinedstorage network list [dimension]` doesn't give any error, and provides the Mean Tick Time and Mean TPS of the controller and keeps updating the numbers upon re-using the command - and the other command tells me the position isn't loaded.
> 
> # TL:DR - Possible Solution
> I found [this issue](https://github.com/TechReborn/RebornStorage/issues/211) on Reborn Storage saying that the mod isn't compatible with RSInfinityBooster. After removing it from my mod list and testing it out I can confirm that this is true.
> 
> You can replace Reborn Storage with the [ExtraStorage](https://www.curseforge.com/minecraft/mc-mods/extrastorage/comments) mod that adds storage disks with increased storage and faster autocrafters with increased pattern slots.



I don't have RSInfinityBooster installed and its not working either.
